homeassistant/components/aprilaire/strings.json

```
@@ -24,6 +24,44 @@
         "name": "Thermostat"
       }
     },
+    "number": {
+      "dehumidification_setpoint": {
+        "name": "Dehumidification setpoint"
+      }
+    },
+    "select": {
+      "air_cleaning_event": {
+        "name": "Air cleaning event",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "event_clean": "Event clean (3 hour)",
+          "allergies": "Allergies (24 hour)"
+        }
+      },
+      "air_cleaning_mode": {
+        "name": "Air cleaning mode",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "constant_clean": "Constant clean",
+          "automatic": "Automatic"
```

Automatic can use a reference key to the other automatic in the file, (I'm on mobile, so I can't type one out for you)

---

Wouldn't that then require referencing a different entity? You would essentially be referencing from fresh_air_mode to the air_cleaning_mode entity. Seems confusing. I would rather keep it as is personally.

---

Yes, and that is no problem because it decreases the amount of unique translation keys
```suggestion
          "automatic": "[%key:components::aprilaire::entity::select::fresh_air_mode::state::automatic%]"
```